### ISSUE TITLE
Can't stop the container?

### ISSUE THREAD

**Issue by lolouk44** [Motion-Project/motion-docker#4]
Just trying the docker container on a fresh Ubuntu 18.04 install to get motion up and running.
Doc is sparse but I've used motion for a few years so was hoping it'd be a smooth transfer.
while testing I needed to restart the container to restart motion. 
I'm getting this message whether I try `docker restart motion` or `docker-compose down`:
```
cctv@cctv:~/Docker/motion$ docker-compose down && docker-compose up -d
Stopping motion ... error

ERROR: for motion  cannot stop container: 7a884c29eec903776054ed7baea2105301e5e249df75625067f280d3798b9ddb: Cannot kill container 7a884c29eec903776054ed7baea2105301e5e249df75625067f280d3798b9ddb: unknown error after kill: docker-runc did not terminate sucessfully: container_linux.go:393: signaling init process caused "permission denied"
: unknown
Removing network motion_default
ERROR: network motion_default id c3decee07a971c15abadeb6da069487f3112d3230c0804bca9e6e4c4262f7039 has active endpoints
```
Any idea?

**Comment by steveneighbour** [comment]
Can you post the compose file you've written?

Is the cctv user in the docker group?  Can you start the container as expected when root/sudo?

**Comment by lolouk44** [comment]
Thanks for the quick reply.
Here is my docker-compose file:
```
  version: '3'
  services:

    motion:
      container_name: motion
      image:  motionproject/motion:latest
      volumes:
        - /home/cctv/Docker/motion/config:/usr/local/etc/motion
        - /home/cctv/Docker/motion/storage:/var/lib/motion
      ports:
        - "8080:8080"
        - "8081:8081"
        - "8082:8082"
        - "8083:8083"
        - "8084:8084"
        - "8085:8085"
        - "8086:8086"
        - "8087:8087"
      restart: unless-stopped
```
cctv is is in the docker group indeed.
If I want to stop the container, I have to do it in these steps:
```
docker-compose down
sudo pkill -9 -f motion
docker-compose down
```
I even tried to log into the container (docker exec -it motion /bin/bash` and run commands like `halt` or `shutdown` but the both fail to execute

**Comment by steveneighbour** [comment]
sorry i didn't see this earlier, hopefully you got it working :(

If not perhaps retry and see how it goes once the open pull request is merged  #5 

i'll fix my notification settings so i see your reply this time..
